src/style.scss

```diff
@@ -118,7 +118,7 @@
 }
 
 .tune-button {
-  @apply rounded-full;
+  @apply h-[42px] rounded-full;
```

Comment:
Any reason for this? It is 42px without this too, 

https://github.com/snapshot-labs/tune/blob/db2f6f25fa29c7af7da25b0a3ba02c8fe8486c9b/src/style.scss#L69